frontend/src/concepts/pipelines/content/__tests__/ComputeRunStatusUtils.spec.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable camelcase*/
+import {
+  BanIcon,
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  NotStartedIcon,
+  QuestionCircleIcon,
+  SyncAltIcon,
+} from '@patternfly/react-icons';
+import React from 'react';
+import { PipelineRunStatusesKF, RunStorageStateKF } from '~/concepts/pipelines/kfTypes';
+import { computeRunStatus } from '~/concepts/pipelines/content/utils';
+
+const run = {
+  created_at: '2023-09-05T16:23:25Z',
+  storage_state: RunStorageStateKF.AVAILABLE,
+  pipeline_spec: {
+    runtime_config: {
+      parameters: {},
+      pipeline_root: '',
+    },
+  },
+  finished_at: '2023-09-05T16:24:34Z',
+  id: 'dc66a214-4df2-4d4d-a302-0d02d8bba0e7',
+  name: 'flip-coin',
+  scheduled_at: '1970-01-01T00:00:00Z',
+  service_account: 'pipeline-runner-pipelines-definition',
+  status: '',
+  error: '',
+  metrics: [],
+};
+
+describe('Pipeline Run status', () => {
```

Comment:
test files should be named the same as the file they are testing. eg `utils.spec.tsx`
You can add specific describe functions to isolate tests pertaining to a specific function.

Reply:
I will update it.